src/tests/ll_verification.py

```diff
@@ -5919,6 +5942,534 @@ def ll_sec_scn_bv_01_c(transport, upperTester, lowerTester, trace):
+        success = getCommandCompleteEvent(transport, upperTester, trace) and (status == 0x00) and success
+
+    return success, initiator, cisConnectionHandle
+
+
+"""
+    LL/CIS/SLA/BV-05-C [Receiving data in Unidirectional CIS]
+"""
+def ll_cis_sla_bv_05_c(transport, upperTester, lowerTester, trace):
+    # Initial Condition
+    #
+    # Connected in the relevant role as defined in the following initial states:
+    #
+    # Note: “default” refers to values specified in Section 4.10.1.3 Default Values for Set CIG Parameters
+    # Commands.
+
+    # Table 4.126: State Variable Values
+    # NOTE: As the IUT is the Slave, the CIG Parameters are those of the Master
+
+    params = SetCIGParameters(
+        SDU_Interval_M_To_S     = 100000, # 100 ms
+        SDU_Interval_S_To_M     = 100000, # 100 ms
+        ISO_Interval            = int(100 // 1.25), # 100 ms
+        NSE                     = 4,
+        Max_PDU_M_To_S          = 247, # TODO: Supposed to be 251
+        Max_PDU_S_To_M          = 0,
+        PHY_M_To_S              = 1,
+        PHY_S_To_M              = 1,
+        FT_M_To_S               = 1,
+        FT_S_To_M               = 1,
+        BN_M_To_S               = 2,
+        BN_S_To_M               = 1, # TODO: Supposed to be 0
+    )
+
+    success, initiator, cisConnectionHandle = state_connected_isochronous_stream_slave(transport, upperTester, lowerTester, trace, params)
+    if not initiator:
+        return success
+
+    for n in range(3):
+        # Create a ISO_SDU of Max_SDU_M_To_S[0] length
+        tx_iso_sdu = tuple([(n + x) % 255 for x in range(params.Max_SDU_M_To_S[0])])
+
+        # Pack the ISO_Data_Load (no Time_Stamp)
+        fmt = '<HH{ISO_SDU_Length}B'.format(ISO_SDU_Length=len(tx_iso_sdu))
+        tx_iso_data_load = struct.pack(fmt, n, len(tx_iso_sdu), *tx_iso_sdu)
+
+        # LT: TX SDU
+        le_iso_data_write(transport, lowerTester, cisConnectionHandle, 2, 0, tx_iso_data_load, 100)
+        success = verifyAndShowEvent(transport, lowerTester, Events.BT_HCI_EVT_NUM_COMPLETED_PACKETS, trace, params.SDU_Interval_M_To_S * 2) and success
+
+        # UT: RX SDU
+        time, handle, pbflags, tsflag, rx_iso_data_load = le_iso_data_read(transport, upperTester, 100)
+        rx_iso_data_load = bytearray(rx_iso_data_load)
+
+        # LT: No RX
+        success = not le_iso_data_ready(transport, lowerTester, 100) and success
+
+        # Unpack ISO_Data_Load
+        rx_offset = 0
+        # a. Get Time_Stamp if present
+        if tsflag:
+            fmt = '<I'
+            (time_stamp,) = struct.unpack_from(fmt, rx_iso_data_load)
+            rx_offset += struct.calcsize(fmt)
+
+        # b. Get Packet_Sequence_Number and ISO_SDU_Length
+        fmt = '<HH'
+        rx_packet_sequence_number, rx_iso_sdu_length = struct.unpack_from(fmt, rx_iso_data_load, rx_offset)
+        rx_offset += struct.calcsize(fmt)
+        rx_iso_sdu_length &= 0xfff # 12 bits valid
```

Comment:
I think it would be worth checking the Packet_Status_Flag in the upper bits of the second half-word.

Reply:
Done